src/components/App.js

```diff
@@ -121,16 +122,14 @@ function App() {
             isChecked={hasSymbols}
             onToggle={setHasSymbols}
           />
-          <label>
-            <input
-              type="number"
-              min="4"
-              max="99"
-              value={length}
-              onChange={e => setLength(+e.target.value)}
-            />
-            Length
-          </label>
+
+          <Range
+            id="length"
+            min={4}
+            max={100}
+            value={length}
+            onChange={e => setLength(parseInt(e.target.value))}
```

Comment:
It would be better if we could just call `setLength` here without all this parsing and `e.target.value`. The callback should receive the actual number on change.